src/SignalR/server/Core/src/HubInvocationContext.cs

```diff
@@ -2,20 +2,41 @@
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
 using System.Collections.Generic;
+using Microsoft.AspNetCore.Authorization;
 
 namespace Microsoft.AspNetCore.SignalR
 {
+    /// <summary>
+    /// A context passed to <see cref="IAuthorizationHandler"/>'s for custom Authorization on Hub methods.
+    /// </summary>
     public class HubInvocationContext
     {
+        /// <summary>
+        /// Instantiates a new <see cref="HubInvocationContext"/> instance.
```

Comment:
```suggestion
        /// Instantiates a new instance of the <see cref="HubInvocationContext"/> class.
```

Is the standard boilerplate AFAICT (https://docs.microsoft.com/en-us/dotnet/api/system.exception.-ctor?view=netcore-2.2)